hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/handlers/ContractHandlers.java

```diff
@@ -49,6 +49,8 @@ public class ContractHandlers {
 
     private final EthereumTransactionHandler ethereumTransactionHandler;
 
+    public static final long MAX_GAS_LIMIT = 15_000_000L;
```

Comment:
Maybe add a comment about how this constant is an "approximation" to the ground truth which is a configuration property somewhere?